### DISCUSSION TITLE
世足賽／AI人工智慧模擬10萬場比賽 推測冠軍是「這國」！

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#867] [General]
最近真的是超級風世足的!不知道大家心中都押甚麼隊伍會贏?現在AI預測也非常熱門，也有人用AI預測縣市長的選舉，雖然AI預測非常強大，但希望大家不要將它拿來做為賭錢的工具為好!
![6363A5D5725381667474901](https://user-images.githubusercontent.com/113499537/204747488-706aaa61-954a-4435-a92d-da750d120fcd.jpeg)
圖片來源:https://www.marieclaire.com.tw/lifestyle/news/69335/fifa-world-cup-qatar-2022

https://udn.com/news/story/7005/6786885

**Comment by Chiway1229** [comment]
雖然說不要賭錢比較好，可是以小搏大的慾望實在難以抵擋

**Comment by Pietra1226** [comment]
等世足賽結束再回來看看是不是很準，只是清一色都沒有亞洲球隊，希望日韓其中一隊至少打到四強QQ

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
真想不到 這屆的世足意想不到的是頻頻發生 日韓兩國都進了陶汰賽

**Comment by HuangHsinTzu** [comment]
AI大概只能透過歷史資料去做推測
真實是裡總是有意想不到的事情發生
(看這屆世足就知道了)
但用AI預測獲勝隊伍確實能夠帶起討論度，讓人用另外一種角度看世足

**Comment by dinosaur1009** [comment]
AI是用科學分析去預判ㄉ 但感覺真實的比賽還有蠻多人為因素影響的

**Comment by ching020721** [comment]
AI畢竟是用過往的大量資料去預測未來，但球賽有趣的地方就是各種異想不到，不過仍可以看看最終結果與AI預測的是否有出入。

**Comment by moonnight3503** [comment]
完了，巴西沒了QAQ

**Comment by StevenWanglolz** [comment]
AI只能跟去過去的數據分析，但是現實世界真的很難預料，有機率發生各種意外，先幫內馬爾哭哭~不推崇賭博的行為

**Comment by Zoe06060606** [comment]
雖然說賭博傷身 但四年一次的世足賽不賭可習啊 小賭怡情就好

**Comment by Evan0979** [comment]
沒在看世足QQ,所以它猜了嗎?好好奇!

**Comment by WEI-CHEN-CHUN** [comment]
賽事預測其實不容易，畢竟有太多變數了，如場地、球員心理等難以量化，因此依靠這部分來進行賭博，不大推

**Comment by Kotori0501** [comment]
我也沒在看世足，因為沒興趣，但系上同學一直在看，我也不知道為啥。
其實我覺得AI只會拿過去的資料進行模擬判斷，因此不一定會很正確的預測出來，因為現實其實挺難預測的，AI僅僅只能做個參考標準而已

**Comment by williamyen0623** [comment]
小賭怡情，大賭郭台銘，我只能說別買太多

**Comment by ruthchen1020** [comment]
然後阿根廷贏了!!
不過我也很想知道這個AI的數據還有根據是甚麼。

**Comment by evanchou1204** [comment]
以結果論來看，他預測的不是很準，不過用機率這種東西來預測未來，應該要長期來看會比較準，如果是預測世足冠軍這樣的一次性預測，我認為AI的預測不會比隨便亂猜一個強隊準多少。

**Comment by CPCpyt** [comment]
近期世足真的是很火紅話題，但我想對AI來說足球的變數還是太大了，要做預測本身就有難度，可以當參考就好。
